index.js

```diff
@@ -1,6 +1,25 @@
 'use strict';
 
 var fnToStr = Function.prototype.toString;
+var reflectApply = typeof Reflect === 'object' && Reflect !== null && Reflect.apply;
+var defineProperty = Object.defineProperty;
+var badArrayLike;
+var isCallableMarker;
+if (typeof reflectApply === 'function' && defineProperty) {
```

Comment:
what about a polyfilled `Reflect.apply` in IE 8, where `defineProperty` doesn't work on non-DOM objects?

Reply:
That’s checked when we try to create the `badArrayLike` object in the next step:

<https://github.com/inspect-js/is-callable/blob/6a0d2a8dba2da7d36e29cc8e8b1bc4e57baadca9/index.js#L10-L14>